crates/s3s-aws/src/body.rs

```diff
@@ -21,12 +21,18 @@ impl Stream for Wrapper {
     type Item = Result<bytes::Bytes, s3s::StdError>;
 
     fn poll_next(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
-        hyper::body::HttpBody::poll_data(Pin::new(&mut self.0), cx)
+        http_body_04::Body::poll_data(Pin::new(&mut self.0), cx)
```

Comment:
https://github.com/smithy-lang/smithy-rs/pull/3380 was merged last week. So we will be able to remove http-body 0.4 after they release a new version.

Reply:
new version got released I updated the implementation
I changed It so that the `s3s::Body` now has an constructor from any `http_body::Body`